t/test-data/Models/hr-inclNOxExtendedWithFilters/PlantProduction.nhd

```diff
@@ -27,9 +27,9 @@ gui      = PlantProduction,Pflanzenbau,Production végétale,Plant production
 
 +nh3_nplantproduction
   format= %.0f
-  print = SummaryPlantProduction,PlantNH3
+  print = SummaryTotal,PlantNH3
   ++labels 
-    sort = 199
+    sort = 1199
```

Comment:
Die Idee, alle Totals in ein "Sub-Report" zusammenzufassen ist eigentlich nicht schlecht. Im aktuellen Modell erscheinen sie allerdings als unterste Zeile der Zusammanfassungen "Tierproduktion" und "Pflanzenbau". Ich würde dies so belassen..

Reply:
Ah, ok, ich dachte, das sei keine Absicht. Werde es zurück ändern.

Reply:
Sonst kann ich das in meinem nächsten PR wieder ändern...

Reply:
Gerne ...